doc/source/python_reference.rst

```diff
@@ -139,7 +139,7 @@ Functions:
    read_bem_solution
    read_bem_surfaces
    read_cov
-   read_dip
+   read_dipoles
```

Comment:
> Fine by me, should be easy enough. It could just have a few attributes, a nice __repr__, and a .plot() method. I assume you're thinking a Dipole is a single dipole with orientation and magnitude as a function of time?

yes exactly. It should be done in another PR